src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
I must be missing something in the sam spec. 

> Chimeric alignment: An alignment of a read that **cannot** be represented as a linear alignment.

What part of the spec are you reading?